pages/notifications.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react'
+import Layout from '~/components/Layout'
+import Meta from '~/components/Meta'
+import ToolbarNotificationItem from '~/components/ToolbarNotifications/ToolbarNotificationsItem'
```

Comment:
```suggestion
import Notification from '~/components/Notification'
```

Muda o arquivo para estar nessa pagina.

Muda para `components/Notification/index.tsx`